cluster/store/retry_schema_with_version.go

```diff
@@ -25,7 +25,7 @@ func (rs retrySchema) ClassInfoWithVersion(ctx context.Context, class string, ve
 	rs.retry(func(s *schema) error {
 		ci = s.ClassInfo(class)
 		if !ci.Exists {
-			return errClassExists
+			return errClassNotFound
```

Comment:
I'm curious here: in which situation do we find the class info but it doesn't exists ? Aka what does `ci.Exists` means in comparison to the `classInfo` simply being absent

Reply:
from what i can tell it's used [here only ](https://github.com/weaviate/weaviate/blob/main/cluster/store/meta_class.go#L40), we can get rid off it later on if it doesn't make sense, I am not aware of it's usage yet. but the error caught my eyes